reference/configuration/framework.rst

```diff
@@ -71,14 +115,65 @@ http_method_override
 .. versionadded:: 2.3
     The ``http_method_override`` option was introduced in Symfony 2.3.
 
-**type**: ``Boolean`` **default**: ``true``
+**type**: ``boolean`` **default**: ``true``
 
 This determines whether the ``_method`` request parameter is used as the intended
 HTTP method on POST requests. If enabled, the
 :method:`Request::enableHttpMethodParameterOverride <Symfony\\Component\\HttpFoundation\\Request::enableHttpMethodParameterOverride>`
-gets called automatically. It becomes the service container parameter named
-``kernel.http_method_override``. For more information, see
-:doc:`/cookbook/routing/method_parameters`.
+gets called automatically. It becomes the service container
+parameter named ``kernel.http_method_override``.
+
+.. seealso::
+
+    For more information, see :doc:`/cookbook/routing/method_parameters`.
+
+.. _reference-framework-trusted-proxies:
+
+trusted_proxies
+~~~~~~~~~~~~~~~
+
+**type**: ``array``
+
+Configures the IP addresses that should be trusted as proxies. It becomes the
+service container parameter called ``kernel.trusted_proxies``.
+
+.. seealso::
+
+    For more details, see :doc:`/cookbook/request/load_balancer_reverse_proxy`.
+
+.. versionadded:: 2.3
+    CIDR notation support was introduced in Symfony 2.3, so you can whitelist whole
+    subnets (e.g. ``10.0.0.0/8``, ``fc00::/7``).
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        framework:
+            trusted_proxies:  [192.0.0.1, 10.0.0.0/8]
+
+    .. code-block:: xml
+
+        <!-- app/config/config.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config trusted-proxies="192.0.0.1, 10.0.0.0/8">
+                <!-- ... -->
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // app/config/config.php
+        $container->loadFromExtension('framework', array(
+            'trusted_proxies' => array('192.0.0.1', '10.0.0.0/8'),
+        ));
```

Comment:
We have a challenge to not create any duplication. Since this is documented elsewhere, I would include the link (you have this) and *not* the code-block. Let's make the *one* place we have this documented really great. I would probably do the same with the CIDR notation, though I think if we just mentioned that in the description, it feels ok (e.g. `... configures the IP addresses (e.g. 192.0.0.1, 10.0.0.0/8)...`). It's not a hard rule, just trying to have less.